bokeh/tests/test_embed.py

```diff
@@ -111,6 +111,13 @@ def test_output_is_without_script_tag_when_wrap_script_is_false(self, mock_make_
         self.maxDiff = None
         self.assertEqual(rawscript.strip(), script_content.strip())
 
+class TestNotebookContent(unittest.TestCase):
+
+    def test_return_type(self):
+        r = embed.notebook_content(_embed_test_plot)
+        self.assertTrue(isinstance(r, tuple))
+        self.assertTrue(len(r), 2)
+
```

Comment:
More can be tested here? The script and idv are both probably amend to aaic t least some b`startwith` testing, or the div could be parsed entirely with beatiful soup and have its structure and expected attributes verified

Reply:
I'll likely refactor most of these tests.